cmd/deploy/start.go

```diff
@@ -21,12 +23,16 @@ import (
 )
 
 const (
-	deployStartShort   = "Start deployment with Armory CD-as-a-Service"
-	deployStartLong    = "Start deployment with Armory CD-as-a-Service"
-	deployStartExample = "armory deploy start [options]"
+	deployStartShort           = "Start deployment with Armory CD-as-a-Service"
```

Comment:
I'm not sure why there's not a merge conflict here: https://github.com/armory-io/armory-cli/blob/main/cmd/deploy/start.go#L23-L26

Reply:
It's because I'm merging this into the other branch that was already approved. Then i'll merge both. It should be a merge conflict on the other PR.